README.md

```diff
@@ -1,2 +1,54 @@
-# datadog-serverless-compat-py
-Datadog Serverless Compatibility Layer for Python
+# Datadog Serverless Compatibility Layer for Python
+
+Datadog library for Python to enable tracing and custom metric submission from Azure Functions and Google Cloud Run Functions (1st gen).
+
+## Installation
+
+1. Install the Datadog Serverless Compatibility Layer.
+```
+pip install datadog-serverless-compat
+```
+
+2. Install the Datadog Tracing Library following the official documentation for [Tracing Python Applications](https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/python).
+
+3. Add the Datadog Serverless Compatibility Layer and the Datadog Tracer in code.
+
+```
+from datadog_serverless_compat import start
+from ddtrace import tracer, patch_all
+
+start()
+patch_all()
+```
+
+## Configuration
+
+1. Set Datadog environment variables
+  - `DD_API_KEY` = `<YOUR API KEY>`
+  - `DD_SITE` = `datadoghq.com`
+  - `DD_ENV` = `<ENVIRONMENT`
+  - `DD_SERVICE` = `<SERVICE NAME>`
+  - `DD_VERSION` = `<VERSION>`
+  - `DD_TRACE_STATS_COMPUTATION_ENABLED` = `true`
```

Comment:
I think you can remove this since it says it is enabled by default in line 38 below

Reply:
Agreed! I removed it